backend/copr_backend/frontend.py

```diff
@@ -8,7 +8,8 @@
 from copr_common.request import SafeRequest, RequestError
 from copr_backend.exceptions import FrontendClientException
 
-MIN_FE_BE_API = 4
+# The frontend counterpart is in `backend_general:send_frontend_version`
+MIN_FE_BE_API = 5
```

Comment:
Further reading this code, I think I was wrong and this bump is not needed?  The frontend/backend changes in this PR are independent, right?

Anyways, the documentation here is very nice, having it (as a separate commit) woudl be nice.

Reply:
I think doing the `MIN_FE_BE_API` bump is a good idea because, at this moment, the backend can't handle a non-200 response. If we start returning 404 and 500 status codes from frontend, the `FrontendClient` will start raising exceptions. At the same time `_get_build_job` doesn't expect them and checks for `if resp.status_code != 200:` instead. I think we need both frontend and backend changes at the same time.